app/controllers/user_controller.py

```diff
@@ -12,7 +12,7 @@ def get_user(user_id: int, username: str) -> Any:
     user = User.get_by_username(username)
     if user:
         is_me = user.user_id == user_id
-        return render_template('account/profile/base.html', user=user, is_me=is_me)
+        return render_template('account/profile/base.html', user=user, is_me=is_me, followers=get_followers(username))
```

Comment:
you dont need to add that, the user object already has a list of its followers `user.followers` which you are using below already
```suggestion
        return render_template('account/profile/base.html', user=user, is_me=is_me)
```